app/uk/gov/hmrc/economiccrimelevyregistration/views/RegistrationSubmittedView.scala.html

```diff
@@ -16,14 +16,15 @@
 
 @import uk.gov.hmrc.economiccrimelevyregistration.config.AppConfig
 @import uk.gov.hmrc.economiccrimelevyregistration.utils.EclTaxYear
+@import uk.gov.hmrc.time.TaxYear
 
 @this(
         layout: templates.Layout,
         govukPanel: GovukPanel,
         appConfig: AppConfig
 )
 
-@(eclReference: String, firstContactEmailAddress: String, secondContactEmailAddress: Option[String])(implicit request: Request[_], messages: Messages)
+@(eclReference: String, firstContactEmailAddress: String, secondContactEmailAddress: Option[String], liabilityYearOption: Option[LiabilityYear])(implicit request: Request[_], messages: Messages)
```

Comment:
liabilityYear would be sufficient.

Prefixing or suffixing parameter names (Hungarian notation is ) with their type is widely panned as a style these days. We know it's an option. The extra text adds no value.